### PR TITLE
Realize home.feature

### PR DESCRIPTION
**Pull request by Mirk32**

### CODE REVIEW

File: features/functionality/home.feature [hunk 1]
```diff
@@ -1,11 +1,19 @@
-@wip
+@ok
 Feature: Home
 
+  Background:
+    Given I am logged in as admin user
+    And I open articles list page
```
Comment (romikoops) [inline features/functionality/home.feature]: And there is article with parameters:
   | attr | value |

File: features/functionality/home.feature [hunk 2]
```diff
@@ -1,11 +1,19 @@
...
   Scenario: visitor can see home page of web application
-    Given there is registered UNIQ_USER user
-    And article created by UNIQ_USER[:email] user with parameters:
-      | title     | UNIQ_ARTICLE[:title]   |
-      | text      | UNIQ_ARTICLE[:text]    |
-    And home page of web application
-    Then I should see UNIQ_ARTICLE[:creation_date] form on home page with data:
-      | article_data | UNIQ_ARTICLE[:creation_time] UNIQ_ARTICLE[:title] |
\ No newline at end of file
+    Given home page of web application
```
Comment (romikoops) [inline features/functionality/home.feature]: When I open home page

File: features/functionality/home.feature [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+    Then I should see Today form on home page with data:
```
Comment (romikoops) [inline features/functionality/home.feature]: What is Today form?

File: features/support/gen.rb [hunk 4]
```diff
@@ -19,11 +19,12 @@ def gen_article_text(serial=nil)
     end
 
     class Article
-      attr_reader :title, :text
+      attr_reader :title, :text, :creation_time
 
       def initialize(params={})
         @title = params.delete(:title)
         @text = params.delete(:text)
+        @creation_time = "#{Time.now.utc.hour+1}#{Time.now.utc.strftime ':%M'}"
```
Comment (romikoops) [inline features/support/gen.rb]: Why we need creation_time? I guess it should add automatically by web app

File: features/functionality/home.feature [hunk 5]
```diff
@@ -1,11 +1,19 @@
...
+      | creation_time | UNIQ_ARTICLE[:creation_time] |
```
Comment (romikoops) [inline features/functionality/home.feature]: I would recommend something like:
Then I should see UNIQ_ARTICLE[:title] article on home page

File: features/support/gen.rb (hunk 4 above)
Comment (Mirk32) [inline features/support/gen.rb]: Because we need to check creation time of article

**Pull request merged**